doc/source/development/extending.rst

```diff
@@ -208,13 +208,201 @@ will
 2. call ``result = op(values, ExtensionArray)``
 3. re-box the result in a ``Series``
 
+:class:`~pandas.api.extensions.ExtensionArray` Series Operations Support
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 0.25.0
+
+In addition to operators like `__mul__` and `__add__`, the pandas Series
+namespace provides a long list of useful operations such as :meth:`Series.round`,
+:meth:`Series.sum`, :meth:`Series.abs`, etc'. Some of these are handled by
+pandas own algorithm implementations (via a dispatch function), while others
+simply call an equivalent numpy function with data from the underlying array.
+In order to support this operations in a new ExtensionArray, you must provide
+an implementation for them.
+
+.. note::
+
+    There is a third category of operation which live on the `pandas`
+    namespace, for example `:meth:pd.concat`. There is an equivalent numpy
+    function `:meth:np.concatenate`, but it is not called by the pandas
+    method. In general, these function should just work with your EA, you do
+    not need to implement more than the general EA interface.
+
+As of 0.25.0, pandas provides its own implementations for the following
```

Comment:
Is listing these out here valuable? Should we instead link to the source where these are defined?

Reply:
The source is a bit opaque for those unfamiliar with it.
https://github.com/pandas-dev/pandas/blob/d47947a79099d650b4bb981a38ce09881ec6f43d/pandas/core/generic.py#L10070-#L10090

Point them to the  docstring of `_reduce` in `ExtensionArray` instead?
https://github.com/pandas-dev/pandas/blob/d47947a79099d650b4bb981a38ce09881ec6f43d/pandas/core/arrays/base.py#L939-#L953

Reply:
Done.